### DISCUSSION TITLE
Can the light source size be changed in forward projection？

### DISCUSSION THREAD

**Discussion by happyzi** [astra-toolbox/astra-toolbox#482] [Q&A] [answered]
Hello! Thank you very much to respond to my questions. 
Could you please tell me if it is possible to adjust the size of the light source or its brightness distribution when performing forward projection to obtain projection images?
In the vone_vec mode, it seems to be a deviation in the position of the point light source. 
Thank you!

**Comment by askorikov** [comment]
Hi! Not directly, since Astra assumes a point source. However,  I imagine you can simulate it by averaging multiple projections from different source positions.

**Comment by Obi-Wan** [comment]
You could do that by computing the associated PSF (point spread function), and then convolving it with the projected images.

**Reply by askorikov** [reply, marked as answer]
Indeed! For cone geometry that wouldn't be an exact solution I guess, but the error should be negligible if your source is compact.
